executor/admin.go

```diff
@@ -393,8 +395,9 @@ func (e *RecoverIndexExec) batchMarkDup(txn kv.Transaction, rows []recoverRows)
 				}
 
 				if handle != rows[i].handle {
-					log.Warnf("[recover-index] The constraint of unique index:%v is broken, handle:%v is not equal handle:%v with idxKey:%v.",
-						e.index.Meta().Name.O, handle, rows[i].handle, key)
+					log.Warn("The constraint of unique index is broken when recovering index",
+						zap.String("index name", e.index.Meta().Name.O), zap.ByteString("key value", key),
```

Comment:
Using `zap.ByteString("key", key)` is better?

Reply:
And should we keep the key as a single word obeying Go style like `indexName` instead of `index name`?

Reply:
Yes, you are right.